src/components/NewBaseCountdown/index.tsx

```diff
@@ -0,0 +1,113 @@
+import { Statistic, StatisticProps } from 'antd';
+import { FontWeightEnum, Typography, ITextProps } from 'aelf-design';
+import { formatCountdown, countdownValueType, FormatConfig, Formatter } from 'antd/lib/statistic/utils';
+import dayjs from 'dayjs';
+import * as React from 'react';
+
+const { Text } = Typography;
+
+const ONE_DAY_IN_MS = 24 * 60 * 60 * 1000;
+
+interface CountdownProps extends StatisticProps {
+  textProps?: ITextProps;
+  value?: countdownValueType;
+  onFinish?: () => void;
+  onChange?: (value?: countdownValueType) => void;
+}
+
+function getTime(value?: countdownValueType) {
+  return new Date(value as any).getTime();
+}
+
+class BaseCountdown extends React.Component<CountdownProps, {}> {
+  countdownId?: number;
```

Comment:
Why use class component here?

Reply:
It will be improved in the next version.